From: Debian Python Team <team+python@tracker.debian.org>
Date: Wed, 12 Jan 2022 18:08:28 -0400
Subject: accept debian version tags

Forwarded: Debian-specific
---
 src/setuptools_scm/git.py | 4 ++++
 1 file changed, 4 insertions(+)

--- setuptools-scm.orig/src/setuptools_scm/git.py
+++ setuptools-scm/src/setuptools_scm/git.py
@@ -254,6 +254,10 @@
     else:
         tag, number_, node = split
         number = int(number_)
+    if tag.startswith('debian/'):
+        tag = tag[7:]
+    if tag.startswith('upstream/'):
+        tag = tag[9:]
     return tag, number, node, dirty
 
 
